kuzzleio/kuzzle-admin-console
# Fix advanced search

## What does this PR do ?
Fixes the advanced search in a pretty hacky way.

### How should this be manually tested?
* create a collection with a `text` and a `keyword` field
* perform searches against each field using the Advanced search
* check you always get results

### Boyscout
Added tests for previous hotfix (on collecion creation)
